w09h01/test/pgdp/pingutrip/kidFriedlyTripTest.java

```diff
@@ -0,0 +1,45 @@
+package pgdp.pingutrip;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.fail;
```

Comment:
```suggestion
import static org.junit.jupiter.api.Assertions.assertArrayEquals;
```

We can use the built-in `assertArrayEquals` method